R/get_datagrid.R

```diff
@@ -1,9 +1,13 @@
 #' Create a reference grid
 #'
 #' Create a reference matrix, useful for visualisation, with evenly spread and
-#' combined values. Usually used to make generate predictions using [get_predicted()].
-#' See this [vignette](https://easystats.github.io/modelbased/articles/visualisation_matrix.html)
+#' combined values. Usually used to make generate predictions using
+#' [get_predicted()]. See this
+#' [vignette](https://easystats.github.io/modelbased/articles/visualisation_matrix.html)
 #' for a tutorial on how to create a visualisation matrix using this function.
+#' \cr\cr
```

Comment:
No need for `\cr` here, markdown should detect empty lines.

Reply:
An empty lines bumps these to the details section.

Reply:
Really? Strange... Anyway, then let's consider as resolved.